modules/ROOT/pages/index-appliance.adoc

```diff
@@ -0,0 +1,62 @@
+= Runtime Fabric Appliance
+
+
+== Runtime Fabric and Other PaaS Providers
+
+Anypoint Runtime Fabric contains all of the components it requires. These components, including Docker and Kubernetes, are optimized to work efficiently with Mule runtimes and other MuleSoft services.
+
+[WARNING]
+Installing Runtime Fabric within an existing Kubernetes-based PaaS is not supported.
+
+If you are already using a PaaS solution, MuleSoft recommends deploying Runtime Fabric in parallel with your PaaS. This enables you to take advantage of the complete benefits of Anypoint Platform.
```

Comment:
I think we can adjust this line to state the user can "consider the self-managed Kubernetes offering."

Reply:
Done